enterprise/__init__.py

```diff
@@ -4,6 +4,6 @@
 
 from __future__ import absolute_import, unicode_literals
 
-__version__ = '0.1.2'
+__version__ = '0.1.3'
```

Comment:
@mattdrayer In this case we'd have to increase MINOR (as we're adding new functionality with each PR), while we're increasing PATCH.